internal/check/handler.go

```diff
@@ -63,6 +63,14 @@ type RESTResponse struct {
 	Allowed bool `json:"allowed"`
 }
 
+// swagger:parameters getCheckRequest
+// nolint:deadcode,unused
+type getCheckRequest struct {
+	// in:query
+	// required: true
```

Comment:
```suggestion
```
It is not required anymore, right?

Reply:
Yeah, and I suppose it's not required for expand either